.github/workflows/test.yml

```diff
@@ -226,7 +226,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: |
-          cd ./clients/swift
```

Comment:
I would prefer not to move the Package.swift file to the top level. With plans for 14 SDKs, the root level of the project could become unwieldy if all clients SDKs did this.